plugins/weather/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/weather/WeatherDarkSkyAPI.java

```diff
@@ -40,31 +47,61 @@
 	private static final String STUTTGART_COORDINATES_LONG = "9.10502";
 
 	private static final String API_SECRET_CONFIG_KEY = "DARKSKY_API_SECRET";
+	
+	private FIODaily dailyReports;
+	private Calendar lastrequest;
```

Comment:
this variable is never assigned, doesn't make sense? Also, `lastRequest` would be nicer